app/scripts/modules/core/src/pipeline/config/pipelineConfigurer.js

```diff
@@ -450,7 +452,7 @@ angular
               this.navigateTo({ section: 'triggers' });
             }
           }
-          $scope.viewState.revertCount++;
+          $scope.viewState.revertCount = ($scope.viewState.revertCount || 0) + 1;
```

Comment:
if revertCount is undefined, `undefined++` results in `revertCount` being `NaN`

`revertCount` is used to re-mount the react components in Triggers.tsx